aws/required-permissions.md

```diff
@@ -129,3 +131,82 @@ Check out the templates directly for more information.
 | `s3:PutObjectTagging` | Action | Put tags on the scanned object. |
 | `sqs:SendMessage` | Action | Send scan messages to the `ScannerQueue`. |
 | [AWSLambdaVPCAccessExecutionRole](https://console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole$jsonEditor?) | Managed Policy | **Optional.** Provides minimum permissions for a Lambda function to execute while accessing a resource within a VPC - create, describe, delete network interfaces and write permissions to CloudWatch Logs. |
+
+## FSS Account Scanner Stack
+
+### Permissions required by account scanner stack
+
+> **Note:** The Permissions for ManagementRole used by Cloud One File Storage Security to maintain the account scanner stack are put into next section.
+
+| Permission | Type | Description |
+| --- | --- | --- |
+| [AWSLambdaBasicExecutionRole](https://console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole$jsonEditor) | Managed Policy | Provides write permissions for a Lambda function to CloudWatch Logs. |
+| `cloudformation:DescribeStacks` | Action | Describe the stacks created by Cloud One File Storage Security. |
+| `events:PutEvents` | Action | **Optional.** Put events from different accounts to the EventBridge default event bus in the same account and region as the account scanner. |
+| `kms:Decrypt` | Action | **Optional.** Decrypt messages from encrypted `ScannerDLQ`, `PostScanActionDLQ`, s3 Bucket, or Data key for scan result encryption. |
```

Comment:
```suggestion
| `kms:Decrypt` | Action | **Optional.** Decrypts messages from encrypted `ScannerDLQ`, `PostScanActionDLQ`, s3 Bucket, or Data key for scan result encryption |
```
The third cell in this row contains a clause phrase, not a full sentence, so doesn't require a period. The verb form should be the same for each clause in this column. Because the implied subject (the managed policy) is singular, the verb should be singular which typically ends in "s".